### PR TITLE
MGMT-12442: Use multi-arch assisted images

### PR DESCRIPTION
**Pull request by adriengentil**
Use the multi-arch installer, controller and agent available on quay.

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -698,6 +698,10 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest-multi
     env:
       ANSIBLE_PLAYBOOK_CREATE_INFRA: equinix_heterogeneous_create_infra_playbook.yml
+      ASSISTED_CONFIG: |
+        AGENT_DOCKER_IMAGE=quay.io/edge-infrastructure/assisted-installer-agent-multi:latest
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:702]: Actually, I think we can build the prow images multi-arch in the first place by reverting this:
https://github.com/openshift/release/pull/34041

It should be better for performance and for having the jobs more portable to other branches

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml (hunk 1 above)
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:702]: I think the problem was that the registry is not multi-arch. We have one URI for x86 and one for arm, which is not what I want here.

Comment (osherdp) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:702]: I see
Then looks good to me

**Review by osherdp — APPROVED** [review]

**Pull request merged**